src/app/authoring-tool/edit-common-advanced/edit-common-advanced.component.html

```diff
@@ -1,34 +1,34 @@
-<edit-component-save-button [componentContent]="componentContent">
+<edit-component-save-button [componentContent]="component.content">
 </edit-component-save-button>
 <br/>
-<edit-component-submit-button [componentContent]="componentContent">
+<edit-component-submit-button [componentContent]="component.content">
 </edit-component-submit-button>
 <edit-component-max-submit
-    *ngIf="componentContent.showSubmitButton"
+    *ngIf="component.content.showSubmitButton"
     [(maxSubmitCount)]="componentContent.maxSubmitCount"
```

Comment:
I think `componentContent` should be changed to `component.content` here. Right now an error is thrown in the components that use the edit-common-advanced component.